src/context/web3.tsx

```diff
@@ -64,28 +68,47 @@ const Web3Provider: React.FC = (props) => {
   const [tokenBalances, setTokenBalances] = useState<any>({});
   const [networkStats, setNetworkStats] = useState<any>({});
   const [lpStats, setLpStats] = useState<any>();
-  const [crucibles, setCrucibles] = useState(
-    [] as {
-      id: string;
-      mintTimestamp: number;
-      balance: string;
-      lockedBalance: string;
-      owner: string;
-      cleanBalance?: string;
-      cleanLockedBalance?: string;
-      cleanUnlockedBalance?: any;
-    }[]
-  );
+  const [crucibles, setCrucibles] = useState([
+    {
+      cleanBalance: 0,
+      initialMistInLP: 0,
+      initialEthInLP: 0,
+      totalLpSupply: 0,
+    },
+  ] as {
+    id: string;
+    mintTimestamp: number;
+    balance: string;
+    lockedBalance: string;
+    owner: string;
+    cleanBalance: any;
+    totalLpSupply: any;
+    cleanLockedBalance?: any;
+    cleanUnlockedBalance?: any;
+    initialMistInLP?: any;
+    initialEthInLP?: any;
+  }[]);
   const [rewards, setRewards] = useState<any>();
   const [onboard, setOnboard] = useState<ReturnType<typeof initOnboard>>(
     null as any
   );
   const [notify, setNotify] = useState<any>(null);
+
+  // GET USER'S LP DEPOSITS
   const { loading, error, data } = useQuery(GET_UNISWAP_MINTS, {
     variables: { userAddress: address },
     skip: !address, // Must have address to query uniswap LP's
   });
 
+  // GET PAIR HISTORY FOR MINTS
+  const [loadPairs, { loading: loadingPairs, data: pairData }] = useLazyQuery(
+    createPairHistoryQuery(
+      "0xcd6bcca48069f8588780dfa274960f15685aee0e",
+      crucibles.length
+        ? crucibles.map((crucible) => crucible.mintTimestamp)
+        : [1615464000]
```

Comment:
Seems like an important timestamp, what does it indicate?

Reply:
It's a fallback, I'll remove in `crucible-frontend`